src/docs/guides/08-how-to-use-google-maps-in-ftw/index.md

```diff
@@ -70,12 +76,14 @@ current location from
 Search for variables: `suggestCurrentLocation` and
 `currentLocationBoundsDistance`.
 
-## 4. Change components: use Google Map versions instead of Mapbox
+---
+
+## 5. Change components: use Google Map versions instead of Mapbox
```

Comment:
Be very careful when changing headings in the site. The anchor links are automatically generated from the headings, so changing a heading also breaks existing links within the site or in support tickets. I see that the numbering was indeed wrong and it probably makes sense to fix it, but just FYI.

Reply:
@kpuputti thanks, very good to know! So 'yarn run format' doesn't re-generate the anchor links? How do they get generated and how to fix them if anchor links are broken?

Reply:
Nothing will automatically fix the links. The anchor is automatically generated from the heading, so if the headings changes, the anchor will change accordingly. There is a plugin that does this automatically.

Internal links are the "easy" part, and there isn't too many of those. I've used https://validator.w3.org/checklink every once in a while to crawl the site for broken links and anchors. But I assume the main problem is links that are not in the site itself, but e.g. in support and other sites.